normits_demand/models/notem/production_models.py

```diff
@@ -419,15 +409,15 @@ def _generate_productions(self,
         pure_hb_prod = nd.get_segmentation_level('notem_hb_productions_pure')
 
         # Reading trip rates
-        du.print_w_toggle("Reading in files...", verbose=verbose)
+        self._logger.info("Reading in files...")
         trip_rates = du.safe_read_csv(
             self.trip_rates_path,
             usecols=self._target_col_dtypes['trip_rate'].keys(),
             dtype=self._target_col_dtypes['trip_rate'],
         )
 
         # ## CREATE THE TRIP RATES DVEC ## #
-        du.print_w_toggle("Creating trip rates DVec...", verbose=verbose)
+        self._logger.info("Creating trip rates DVec...")
```

Comment:
You can remove both of these messages - they're too granular and not needed anymore